example/lib/meeting/meeting_store.dart

```diff
@@ -256,18 +251,27 @@ abstract class MeetingStoreBase extends ChangeNotifier
   @override
   void onJoin({required HMSRoom room}) async {
     hmsRoom = room;
+    if(room.hmshlsStreamingState?.running??false){
+      hasHlsStarted=true;
+      streamUrl = room.hmshlsStreamingState?.variants[0]?.hlsStreamUrl??"";
```

Comment:
what happens if streamUrl is Empty ?

Reply:
If running is true then why stream url would be empty?